android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/Keyboard.java

```diff
@@ -156,16 +157,21 @@ public JSONObject toJSON() {
   }
 
   // Default sil_euro_latin keyboard
-  public static final Keyboard DEFAULT_KEYBOARD = new Keyboard(
-    KMManager.KMDefault_PackageID,
-    KMManager.KMDefault_KeyboardID,
-    KMManager.KMDefault_KeyboardName,
-    KMManager.KMDefault_LanguageID,
-    KMManager.KMDefault_LanguageName,
-    KMManager.KMDefault_KeyboardVersion,
-    null, // will use help.keyman.com link because context required to determine local welcome.htm path,
-    "",
-    false,
-    KMManager.KMDefault_KeyboardFont,
-    KMManager.KMDefault_KeyboardFont);
+  public static Keyboard getDefaultKeyboard(Context context) {
+    String version = KMManager.getLatestKeyboardFileVersion(
```

Comment:
I can't help but notice that this is the only reference to `getLatestKeyboardFileVersion` I see within this PR's file changes.  How is this being updated at build time to match the downloaded resource?

Reply:
When MainActivity installs the default keyboard (l. 203)

> KMManager.addKeyboard(this, Keyboard.getDefaultKeyboard(...)

at runtime, the keyboard version is retrieved (from kmp.json).